aten/src/ATen/native/native_functions.yaml

```diff
@@ -1,5 +1,29 @@
 # See README.md in this directory for more guidance
 
+- func: _cast_uint8_t(Tensor self, bool non_blocking=false) -> Tensor
+  variants: function, method
+
+- func: _cast_int8_t(Tensor self, bool non_blocking=false) -> Tensor
+  variants: function, method
+
+- func: _cast_double(Tensor self, bool non_blocking=false) -> Tensor
+  variants: function, method
+
+- func: _cast_float(Tensor self, bool non_blocking=false) -> Tensor
+  variants: function, method
+
+- func: _cast_int(Tensor self, bool non_blocking=false) -> Tensor
+  variants: function, method
+
+- func: _cast_int64_t(Tensor self, bool non_blocking=false) -> Tensor
+  variants: function, method
+
+- func: _cast_int16_t(Tensor self, bool non_blocking=false) -> Tensor
+  variants: function, method
+
+- func: _cast_Half(Tensor self, bool non_blocking=false) -> Tensor
```

Comment:
Capitalized?

Reply:
Yeah, that's how it's named in ScalarTypehttps://github.com/pytorch/pytorch/blob/9f980b179506489d226051259eacec8639c00c5f/aten/src/ATen/ScalarType.h#L19